client/driver/driver_test.go

```diff
@@ -17,7 +17,7 @@ var basicResources = &structs.Resources{
 	MemoryMB: 256,
 	Networks: []*structs.NetworkResource{
 		&structs.NetworkResource{
-			IP:            "1.2.3.4",
+			IP:            "0.0.0.0",
```

Comment:
Docker has to be able to bind to this in order for tests to pass. Without doing some weird hacks to get the IP for a remote docker daemon, our other option is `127.0.0.1`.